utils/model_uploader/sparse_model_autotracing.py

```diff
@@ -0,0 +1,460 @@
+        license_var_filepath = OUTPUT_DIR + "/" + LICENSE_VAR_FILE
+        with open(license_var_filepath, "w") as f:
+            f.write(str(license_verified))
+    except Exception as e:
+        print(
+            f"Cannot store license_verified ({license_verified}) in {license_var_filepath}: {e}"
+        )
+
+
+def store_description_variable(config_path_for_checking_description: str) -> None:
+    """
+    Store model description in OUTPUT_DIR/DESCRIPTION_VAR_FILE
+    to be used to generate issue body for manual approval
+
+    :param config_path_for_checking_description: Path to config json file
+    :type config_path_for_checking_description: str
+    :return: No return value expected
+    :rtype: None
+    """
+    try:
+        os.makedirs(OUTPUT_DIR, exist_ok=True)
+        description_var_filepath = OUTPUT_DIR + "/" + DESCRIPTION_VAR_FILE
+        with open(config_path_for_checking_description, "r") as f:
+            config_dict = json.load(f)
+            description = (
+                config_dict["description"] if "description" in config_dict else "-"
+            )
+        print(f"Storing the following description at {description_var_filepath}")
+        print(description)
+        with open(description_var_filepath, "w") as f:
+            f.write(description)
+    except Exception as e:
+        print(
+            f"Cannot store description ({description}) in {description_var_filepath}: {e}"
+        )
+
+
+def main(
+    model_id: str,
+    model_version: str,
+    tracing_format: str,
+    model_description: Optional[str] = None,
+) -> None:
+    """
+    Perform model auto-tracing and prepare files for uploading to OpenSearch model hub
+
+    :param model_id: Model ID of the pretrained model
+    :type model_id: string
+    :param model_version: Version of the pretrained model for registration
+    :type model_version: string
+    :param tracing_format: Tracing format ("TORCH_SCRIPT", "ONNX", or "BOTH")
+    :type tracing_format: string
+    :param model_description: Model description input
+    :type model_description: string
+    :return: No return value expected
+    :rtype: None
+    """
+
+    print(
+        f"""
+    === Begin running model_autotracing.py ===
+    Model ID: {model_id}
+    Model Version: {model_version}
+    Tracing Format: {tracing_format}
+    Model Description: {model_description if model_description is not None else 'N/A'}
+    ==========================================
+    """
+    )
+
+    # Now Sparse model auto tracing only support Torch Script.
+    assert (
+        tracing_format == TORCH_SCRIPT_FORMAT
+    ), f"Now Only {TORCH_SCRIPT_FORMAT} is supported."
+
+    ml_client = MLCommonClient(OPENSEARCH_TEST_CLIENT)
+    pre_trained_model = SparseEncodingModel(model_id)
+    original_encoding_data = pre_trained_model.process_queries(TEST_SENTENCES)
+    pre_trained_model.save(path=TEMP_MODEL_PATH)
+    license_verified = verify_license_in_md_file()
```

Comment:
`SparseEncodingModel.save` just call transformers model save and tokenizer save; it doesn't save license file to TEMP_MODEL_PATH. How can we verify the license in TEMP_MODEL_PATH?